ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml

```diff
@@ -44,6 +44,7 @@ tests:
   steps:
     cluster_profile: aws-sd-qe
     env:
+      NAME_PREFIX: pr-rosa-hcp
```

Comment:
We do not need to name the different prefixes for each presubmit job; just the same one, e.g., `pr-` is enough.

Reply:
Updated